src/cli.ts

```diff
@@ -50,10 +51,21 @@ const logToolbeltVersion = () => {
   log.debug(`Toolbelt version: ${pkg.version}`)
 }
 
+const validToken = (): boolean => {
+  const token = getToken()
+  if (!token) { return false }
+
+  const decoded = decode(token)
+  if (decoded === null || typeof decoded === 'string' || decoded.exp === undefined) { return false }
+  if (Number(decoded.exp) < (Date.now() / 1000)) { return false }
```

Comment:
Is this using the same timezone? `Date.now()` probably returns the time in the current timezone while the timestamp in the token is probably UTC. Could you make sure there's no mismatch here?

Reply:
According to the JWT spec and the Date.Now() doc, both should be calculated using the Unix epoch, which is UTC